assets/js/util/test/getReAuthUrl.js

```diff
@@ -0,0 +1,62 @@
+/**
+ * Internal dependencies
+ */
+import { getReAuthURL } from '../';
+
+const createSiteKit = ( apiKey ) => {
```

Comment:
A bit unrelated, but let's remove the `apiKey` references here entirely as they were only necessary when we actually used one.